v1.1/show-jobs.md

```diff
@@ -4,35 +4,77 @@ summary: The SHOW JOBS statement lists all currently active schema changes and b
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> The `SHOW JOBS` [statement](sql-statements.html) lists details about currently active schema changes and backup/restore jobs, including:
+<span class="version-tag">New in v1.1:</span> The `SHOW JOBS` [statement](sql-statements.html) lists all of the types of long-running tasks your cluster has performed, including:
 
-- xx
-- xx
-- xx
+- Schema changes through `ALTER TABLE`
+- Enterprise [`BACKUP`](backup.html), [`RESTORE`](restore.html), and `IMPORT`
 
-These details let you...
+These details can help you understand the status of crucial tasks that can impact the performance of your cluster, as well as help you control them.
 
 <div id="toc"></div>
 
 ## Required Privileges
 
-XX
+By default, only the `root` user can execute `SHOW JOBS`.
 
 ## Synopsis
 
 {% include sql/{{ page.version.version }}/diagrams/show_jobs.html %}
 
 ## Response
 
-The following fields are returned for each session:
+The following fields are returned for each job:
 
 Field | Description
 ------|------------
+`id` | A unique ID to identify each job. This value is used if you want to control jobs (i.e., [pause](pause-job.html), [resume](resume-job.html), or [cancel](cancel-job.html) it).
+`type` | Describes the job as `SCHEMA CHANGE`, [`BACKUP`](backup.html), [`RESTORE`](restore.html), or `IMPORT`.
+`description` | The command that started the job.
+`username` | The user who started the job.
+`status` | Describes the job's current state as `pending`, `running`, `paused`, `failed`, `succeeded`, or `canceled`.
```

Comment:
nit: My OCD language impulse wants this description to take the same form as the others, e.g.:

> The job's current state. Possible values: ...

Reply:
Done.